images/bootstrap/runner.sh

```diff
@@ -29,7 +29,8 @@ cleanup_dind() {
     if [[ "${DOCKER_IN_DOCKER_ENABLED:-false}" == "true" ]]; then
         echo "Cleaning up after docker"
         docker ps -aq | xargs -r docker rm -f || true
-        service docker stop || true
+        echo "Waiting for docker to stop for 30 seconds"
+        timeout 30 service docker stop || true
```

Comment:
`service docker stop` is already exiting though?

Reply:
`timeout` would terminate `service docker stop` if it hangs, but what we're actually seeing in the logs is some internal failure attempting to kill dockerd?

Reply:
it was exiting, but i think it took forever (looking at timestamps around it).

I am happy to try something else as well if this is not kosher.

Reply:
Ahh! Got it.

Reply:
We should also do this in images/krte/wrapper.sh

/lgtm
/approve
/hold

Reply:
good call @BenTheElder, updated wrapper.sh now.